### PR TITLE
Add Tail Rotor

### PR DESCRIPTION
**Pull request by BradMick**

### CODE REVIEW

**Review by Ollieollieolllie — APPROVED** [review]
looks good to me

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/cfgFunctions.hpp [hunk 1]
```diff
@@ -9,6 +9,14 @@ class CfgFunctions
     class fza_ah64_project_sfmplus
     {
         tag = "FZA_sfmplus";
+        class actuator {
```
Comment (Ollieollieolllie) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/cfgFunctions.hpp:12]: Not related to this pr

**Review by Ollieollieolllie — CHANGES_REQUESTED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotorTail.sqf [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+                    \nBlade Tip Vel = %2
+                    \nRotor Power Req = %3 kW
+                    \nRotor Torque = %4 Nm 
+                    \nE1 Tq = %5 % E2 Tq = %6 %
+                    \nVelZ = %7
+                    \nInduced Vel Scalar = %8
+                    \nGnd Eff Scalar = %9
+                    \nStab = %10
+                    \nPitch = %11", _rtrOmega, _bladeTipVel, _rtrPowerReq * 0.001, _reqEngTorque, (_reqEngTorque / 2) / 481, (_reqEngTorque / 2) / 481, _velZ, _inducedVelocityScalar, _gndEffScalar, fza_sfmplus_collectiveOutput, _heli call BIS_fnc_getPitchBank select 0];
+                    */
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotorTail.sqf:134]: We should look at ways to share a lot of the info between simpleRotorMain and simpleRotorTail - though leaving for now

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/cfgFunctions.hpp (hunk 1 above)
Comment (mattysmith22) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/cfgFunctions.hpp:12]: Agreed, these need to be removed as these functions do not exist in this PR

**Review by mattysmith22 — CHANGES_REQUESTED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/cfgFunctions.hpp (hunk 1 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/cfgFunctions.hpp:12]: Fixed

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotorMain.sqf [hunk 3]
```diff
@@ -112,9 +113,10 @@ private _torqueZ      = _axisZ vectorMultiply ((_rtrTorque  * _rtrTorqueScalar)
 //Rotor thrust force
 _heli addForce [_heli vectorModelToWorld _thrustZ, _rtrPos];
 //Main rotor torque effect
-_heli addTorque (_heli vectorModelToWorld _torqueZ);
-
-//Camera shake effect for ETL
+if (fza_ah64_sfmplusEnableTorqueSim) then {
```
[the remark this answers is not in the document]
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotorMain.sqf:116]: Removed for now...

**Review by Ollieollieolllie — APPROVED** [review]
Looks Great

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotorTail.sqf (hunk 2 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotorTail.sqf:134]: About the only consistent thing are rotor parameters, there are some pretty significant differences with regards to implementation however

**Review by mattysmith22 — APPROVED** [review]

**Pull request merged**